jujugui/static/gui/src/app/views/topology/relation.js

```diff
@@ -489,11 +489,8 @@ YUI.add('juju-topology-relation', function(Y) {
      */
     mousemove: function(d, self) {
       if (self.clickAddRelation) {
-        var mouse = d3.mouse(self.get('dragplane'));
-        var box = self.get('addRelationStart_service');
-        d3.event.x = mouse[0];
```

Comment:
It appears that the mouseEvents are now read-only so these fail to be assigned. They don't appear to have any negative side effects when removed. Originally it may have been used for panning synchronization according to @makyo